components/telemetry-operator/controllers/logpipeline_controller_test.go

```diff
@@ -51,13 +51,15 @@ var _ = Describe("LogPipeline controller", func() {
     Emitter_Mem_Buf_Limit 10M
 
 [FILTER]
-    Name   grep
-    Match   *
-    Regex   $kubernetes['labels']['app'] my-deployment
+    match *
```

Comment:
This should be possible, right? 
But since this test doesn't valide the config with the webhook, it is unnoticed. What should we do?

Reply:
What do you mean? The lower-case `match` or indentation? I guess both should work.

Reply:
I mean the `*` (match all) expression. It works since the validation webhook is not deployed in this test.